go/interpreter/evmc/evmc_interpreter.go

```diff
@@ -166,7 +166,13 @@ type hostContext struct {
 }
 
 func (ctx *hostContext) AccountExists(addr evmc.Address) bool {
```

Comment:
Aren't there any functions which use this function as `AccountExists` and not as `AccountEmpty`?

Reply:
As we learned lately, nowhere within an interpreter information about the existence of an account is needed. So nobody on the other side of the EVMC interface should need account existence information. 

Account existence information seems to be only relevant on the processor level.